powersimdata/network/constants/region/usa.py

```diff
@@ -175,6 +173,7 @@ def get_state_mapping(zone):
 
     mapping["state"] = set(zone["state"])
     mapping["abv"] = set(zone["abv"])
+    mapping["state_abbr"] = set(zone["abv"])
```

Comment:
Do we need the same entry twice?

Reply:
It kind of help in my refactoring of the `area_to_loadzone` function and functions that check that areas exist in a grid model that I do in `ben/eu` since one of the area type a user can pass to the function is `state_abbr` and `country_abbr`. See https://github.com/Breakthrough-Energy/PowerSimData/blob/d232cc0f5cb0965c00045127e3bddc7e4c4417b4/powersimdata/network/model.py#L43 for instance